qemu/tests/fio_perf.py

```diff
@@ -136,7 +108,22 @@ def fio_thread():
         """
         run fio command in guest
         """
-        session.cmd_status(run_cmd, cmd_timeout)
+        fio.run(run_fio_options)
```

Comment:
It is better to make run/clean in a try-finally blocker.

I would like to suggest:
```
fio = generate_instance(params, vm, 'fio')
try:
    fio.run(run_fio_options)
finally:
    fio.clean()
```

In this way, clean the result file in guest os can be a separated step as before.